### PR TITLE
feat(volume-backup): detach a volume

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-7636-volume-backup`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-10614
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. detach a volume from instance to be able to create a volume backup

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.controller.js [hunk 1]
```diff
@@ -46,7 +63,12 @@ export default class VolumeBackupCreateController {
       volumeOption: selectedVolumeType,
     } = this.volumeBackupModel.selected;
 
-    return selectedVolume && selectedVolumeType && this.volumeBackupModel.name;
+    return (
+      selectedVolume &&
+      selectedVolume.attachedTo.length === 0 &&
+      selectedVolumeType &&
+      this.volumeBackupModel.name
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.controller.js:70]: **nitpick:** The expression can be simplified.
```suggestion
      selectedVolume?.attachedTo.length === 0 &&
      selectedVolumeType &&
      this.volumeBackupModel.name
```

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/detach-volume.controller.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    return this.volumeBackupService
+      .detachVolumeFromInstance(
+        this.projectId,
+        this.volume.id,
+        this.instance.id,
+      )
+      .then(() => {
+        this.volume.attachedTo = [];
+      })
+      .then(() => this.goBack())
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/detach-volume.controller.js:33]: **nitpick:** Unnecessary .`then()`
```suggestion
      .then(() => {
        this.volume.attachedTo = [];
        return this.goBack();
      })
```

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.controller.js:70]: done

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/detach-volume.controller.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/detach-volume.controller.js:33]: done

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_storages_volume_backup_create_detach_volume_title": "Détacher votre volume {{ volumeName }} à son instance {{ instanceName }}",
+  "pci_projects_project_storages_volume_backup_create_detach_volume_description": "souhaitez vous détacher votre volume de son instance ?",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/translations/Messages_fr_FR.json:3]:
```suggestion
  "pci_projects_project_storages_volume_backup_create_detach_volume_description": "Souhaitez-vous détacher votre volume de son instance ?",
```

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/translations/Messages_fr_FR.json:3]: Nice 👍 , i will change it

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/detach-volume/translations/Messages_fr_FR.json]: Changed

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**